index.php

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-<link rel="shortcut icon" href="https://yeolom.com//favicon.png" type="image/x-icon"> 
+<link rel="shortcut icon" href="\yeolom2021-2022May\favicon.png" type="image/x-icon"> 
```

Comment:
So,  we would need to refactor the code because we can't add the folder to it.

But it's all good.